### DISCUSSION TITLE
404 error

### DISCUSSION THREAD

**Discussion by oanaciuca** [community/community#131344] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello GitHub Support Team,

I am experiencing an issue with my GitHub Pages deployment. My site is not accessible, and I receive a 404 error at https://oanaciuca.github.io/print-it/. The repository structure is correctly set up, and the site works perfectly in local environments.

Details:

Repository: [oanaciuca/print-it](https://github.com/oanaciuca/print-it)
Branch: main
Configuration: GitHub Pages is set to deploy from the main branch with the root folder.
I have tried the following troubleshooting steps:

Verified that index.html is at the root of the repository.
Ensured all resource paths are correct and match the case sensitivity.

Cleared browser cache and tried accessing directly via https://oanaciuca.github.io/print-it/index.html, https://oanaciuca.github.io/print-it/assets/style.css, and https://oanaciuca.github.io/print-it/assets/script.js.
Despite these steps, the site remains inaccessible with a 404 error. Could you please assist me in resolving this issue?

Thank you for your support.

Best regards,

oana ciuca

**Comment by davevad93** [comment, marked as answer]
Hi @oanaciuca ,

I've checked your repo and the problem is that the structure isn't correct, in the root you have a folder called `Print-it-JS-main` with inside the required files, delete that folder and put the files at the root of the repo. The right structure is as follows:

/print-it
│
├── assets
├── README.md
└── index.html
